### DISCUSSION TITLE
How would you centralize information obtained by an audit script ran an all your devices ?

### DISCUSSION THREAD

**Discussion by pierrecorsini** [Ylianst/MeshCentral#5812] [Q&A]
Hi

I do have a scripts thats does some audit check such as : check that specific softs are up to date, check the number of errors in the windows event log, check %disk used.

The script basically returns a JSON text with the resulting data.

I am running it device-per-device, or server-wide using the ScriptTask plugin. The issue is that I am not able to centralize all the results in one place/document/DB.

Would you have any advice ?
The idea option would be to have the audit result displayed into MC (like AV/bitlocker info on device list) but this is not a feature.
I'm considering running a "audit result server" that would be used by each node to communicate it's results... But that seems wrong.

**Comment by si458** [comment]
Meshcentral was never designed as a remote management software (rmm) 
It was designed as a remote support software!
Hence people made third party plugins like `ScriptTask` but they aren't perfect
Sadly the is no central database to store information in I'm afraid

**Comment by silversword411** [comment]
RMM like Tactical RMM can do that
https://docs.tacticalrmm.com/

**Comment by pierrecorsini** [comment]
Yes a pure RMM would exactly do that, but I'm not planning to change my setup and MC with the ScriptTask plugin is fine for me 😅

P.S. Adding a way to export script exec logs on the [ScriptTask](https://github.com/ryanblenis/MeshCentral-ScriptTask) plugin might be the simplest, I'll have a look at the code.
